src/libraries/System.Text.Json/src/System/Text/Json/Document/JsonElement.cs

```diff
@@ -454,10 +456,10 @@ public string GetString()
 
             if (_parent is JsonDocument document)
             {
-                return document.GetString(_idx, JsonTokenType.String);
+                return document.GetString(_idx, JsonTokenType.String)!;
```

Comment:
I don't think this is correct. `document.GetString` could return null and we shouldn't bang this away. In fact, `JsonElement.ToString()` should be changed to return `string?`

Reply:
It's actually JsonElement.GetString that can return null (as well as JsonElement.ToString).  There is no JsonDocument.GetString :smile:.

Reply:
I thought it will not be null as JsonTokenType is String 😛, good catch

Reply:
> There is no JsonDocument.GetString 

What do you mean? The source code I commented on has `document.GetString`. We have an internal method on JsonDocument called `GetString`
https://github.com/dotnet/runtime/blob/743dc49e75e953f96cbadd3901562e030b227293/src/libraries/System.Text.Json/src/System/Text/Json/Document/JsonDocument.cs#L246-L247

Reply:
Ah, right.  Sorry, pre-coffee thinking (I was only thinking of the ref.cs, and therefore public methods).